spec/acceptance/checking_monitored_resources_spec.rb

```diff
@@ -24,9 +24,11 @@
 
   context 'when another service monitors the same link' do
     let(:other_resource_monitor) do
-      LinkMonitor::CreateResourceMonitor.new(
+      LinkMonitor::UpsertResourceMonitor.new(
         links: resource_monitor.links.map(&:uri),
-        service: "local-link-manager"
+        service: "local-link-manager",
```

Comment:
Incredibly small thing! But, might be confusing when doing a search through the code, so this should be `local-links-manager`.

Reply:
👍